dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="A library that implement a layer on top of PKCS#11 API"
+DESCRIPTION="The official header files for PKCS#11 Standard were released \
+under a non-free license. libp11 does not include these header files, but \
+instead uses a rewritten pkcs11.h header file from the Scute project."
+HOMEPAGE="https://github.com/opensc/libp11"
+COPYRIGHT="1991, 1999 Free Software Foundation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/releases/download/libp11-0.4.3/libp11-0.4.3.tar.gz"
+CHECKSUM_SHA256="938bf73fc57ef8b543c6a2ceab617d7d83ccf60456241b73a61adb5ca37cd7c5"
+SOURCE_DIR="libp11-$portVersion"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
We will need to change SECONDARY_ARCHITECTURES (and maybe ARCHITECTURES too) but this will depend on which architectures the recipe builds or fails to build. So I'll update this comment later.